### DISCUSSION TITLE
[WinError 1314] A required privilege is not held by the client

### DISCUSSION THREAD

**Discussion by glmontano** [pyannote/pyannote-audio#1335] [Q&A]
Good day!

I'm using `from pyannote.audio import Pipeline` and `pipeline = Pipeline.from_pretrained("pyannote/speaker-diarization", use_auth_token=ACCESS_TOKEN)` to diaritize audio.

The error I receive is 

```python
  File "C:\dev\python38\lib\pathlib.py", line 1361, in symlink_to
    self._accessor.symlink(target, self, target_is_directory)
OSError: [WinError 1314] A required privilege is not held by the client: 'C:\\Users\\ME\\.cache\\huggingface\\hub\\models--speechbrain--spkrec-ecapa-voxceleb\\snapshots\\5c0be3875fda05e81f3c004ed8c7c06be308de1e\\hyperparams.yaml' -> 'C:\\Users\\ME\\.cache\\torch\\pyannote\\speechbrain\\hyperparams.yaml'
```
How may I solve this **without** admin privlidges?

Thank you.

**Comment by kaixxx** [comment]
Your problem sounds familiar to me... Take a look at this pull request and my comment on it: https://github.com/pyannote/pyannote-audio/pull/1359 This solved it for me.
